README.md

```diff
@@ -35,9 +35,33 @@ fetchBodyTypes()
 | `optionServiceUrl`           | URL to [Options Service](https://option-service.preprod.carforyou.ch/swagger-ui/index.html)                         |
 | `analyticsServiceUrl`        | URL to [Analytics Service](https://carforyou-analytics-service.preprod.carforyou.ch/swagger-ui/index.html)          |
 | `userNotificationServiceUrl` | URL to [User Notification Service](https://user-notification-service.preprod.carforyou.ch/swagger-ui/index.html)    |
-| `tokenRefreshServiceUrl`     | URL to Auth Service used to refresh access tokens                                                                   |
 | `debug`                      | Set to `true` to `console.log` requests and API responses.                                                          |
 
+### Authenticated Requests
+
+Authenticated requests need to be handled in a different way depending on weather the request is performed server- or client-side.
+Client-Side, we can use the api client as a singelton to store the token. Server-side the access token needs to be passed to the request in order
```

Comment:
```suggestion
Client-Side, we can use the api client as a singleton to store the token. Server-side the access token needs to be passed to the request in order
```

I was involved in this decision, but reading this, I'm thinking - but why! Is it worth putting the mental overhead on the consumer to decide which one is the right approach for what context?
How much worse would it be for a server-side consumer if they'd have to pass the token with every call as well? The app can still decide to store the token in some singleton-ish way